tests/integration/targets/openssh_keypair/tasks/impl.yml

```
@@ -0,0 +1,557 @@
+---
+####################################################################
+# WARNING: These are designed specifically for Ansible tests       #
+# and should not be used as examples of how to write Ansible roles #
+####################################################################
+
+  # Ensures no conflicts from previous test runs
+- name: Cleanup Output Directory
+  ansible.builtin.file:
+    path: "{{ item }}"
+    state: absent
+  with_fileglob:
+    - "{{ output_dir }}/privatekey*"
+    - "{{ output_dir }}/regenerate*"
+
+- name: Generate privatekey1 - standard (check mode)
```

For easier debugging, it can be very useful to include the backend's name in very task's name. (Best to use a regexp replace to add it everywhere ;-) )
```suggestion
- name: "({{ backend }}) Generate privatekey1 - standard (check mode)"
```